src/Swarm/Game/Terrain.hs

```diff
@@ -59,9 +59,9 @@ getTerrainWord = T.toLower . T.pack . init . show
 terrainMap :: Map TerrainType Display
 terrainMap =
   M.fromList
-    [ (StoneT, defaultTerrainDisplay '▒' (ATerrain "stone"))
-    , (DirtT, defaultTerrainDisplay '▒' (ATerrain "dirt"))
-    , (GrassT, defaultTerrainDisplay '▒' (ATerrain "grass"))
+    [ (StoneT, defaultTerrainDisplay ' ' (ATerrain "stone"))
```

Comment:
I was going to suggest that we remove the first `Char` argument from the `defaultTerrainDisplay` function.  But I suppose it could still be useful to define terrain types in the future which are displayed as some non-space character?

Reply:
Agreed it should be removed.